x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -177,8 +177,27 @@ export const InnerFieldItem = function InnerFieldItem(props: FieldItemProps) {
     field,
     indexPattern.id,
   ]);
+  const lensFieldIcon = <LensFieldIcon type={field.type as DataType} />;
+  const lensInfoIcon = (
+    <EuiIconTip
+      anchorClassName="lnsFieldItem__infoIcon"
+      content={
+        hideDetails
+          ? i18n.translate('xpack.lens.indexPattern.fieldItemTooltip', {
+              defaultMessage: 'Drag and drop to visualize.',
+            })
+          : i18n.translate('xpack.lens.indexPattern.fieldStatsButtonLabel', {
+              defaultMessage: 'Click for a field preview, or drag and drop to visualize.',
+            })
+      }
+      type="iInCircle"
+      color="subdued"
+      size="s"
+    />
+  );
   return (
     <EuiPopover
+      ownFocus
```

Comment:
Is ownFocus necessary for Lens? when I navigate with tab keys, press return, i can't continue navigate the field List, need to use Esc key to return. While in Discover there are elements that I could navigate to in the popover, I don't see such element in Lens

Reply:
We should be using `ownFocus` on all popovers and flyouts. Though there's nothing to activate in the popover (at least, right now), `ownFocus` is also currently the only way for a screenreader users to be moved to the popover otherwise they have no idea that something changed.

There are other possible ways to enable this without `ownFocus` but none that are already available to us in either Kibana or EUI.